acceptance-tests/build.gradle

```diff
@@ -39,5 +40,7 @@ task acceptanceTest(type: Test) {
   group = 'verification'
 
   useJUnitPlatform()
+  // toggle to show standard out and standard error of the test JVM(s) on the console
+  testLogging.showStandardStreams = (System.getenv("AT_DEBUG") != null)
```

Comment:
is this something we do elsewhere?

Reply:
No, this was an attempt to modify the env variable via CircleCI UI ... since that didn't worked, I've reverted it back to literal false after debug run.